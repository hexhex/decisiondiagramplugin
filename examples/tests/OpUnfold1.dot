digraph {
     v2 -> leaf2 [label="else"];
     v1 -> leaf2_1 [label="else"];
     root -> v2 [label="else"];
     v2 -> leaf1 [label="z<w"];
     v1 -> leaf1_1 [label="z<w"];
     root -> v1 [label="x<y"];
     leaf2_1 [label="leaf2 [class2]"];
     leaf1_1 [label="leaf1 [class1]"];
     leaf2 [label="leaf2 [class2]"];
     leaf1 [label="leaf1 [class1]"];
     v2 [label="v2"];
     v1 [label="v1"];
     root [label="root"];
}
