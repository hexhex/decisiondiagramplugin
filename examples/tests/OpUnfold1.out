{elseedge(v2,leaf2), elseedge(v1,leaf2_1), elseedge(root,v2), conditionaledge(v2,leaf1,z,"<",w), conditionaledge(v1,leaf1_1,z,"<",w), conditionaledge(root,v1,x,"<",y), leafnode(leaf1_1,class1), leafnode(leaf2_1,class2), leafnode(leaf2,class2), leafnode(leaf1,class1), innernode(v2), innernode(v1), innernode(root), root(root)}
